### ISSUE TITLE
Server closed the connection after character selection

### ISSUE THREAD

**Issue by Timorem** [Emudofus/BehaviorIsManaged#58]
I had this problem with only few characters, I have no error log and the server closed the connection by itself... I don't know why

Has anybody the same issue ?

**Comment by FastFrench** [comment]
Yes, and no idea why. When it happens, I connect the character without BiM, remove the weapon (if it's not enough then remove all equipement) and move the character to another map. Then I can connect through BiM, and you can put stuff back. 
Even with exceptions active, no hint on what's wrong.

**Comment by Timorem** [comment]
That's weird, and if you reconnect again with the equipped stuff does it crash ?
It certainly comes from Inventory class, must look at it.

**Comment by FastFrench** [comment]
When you cannot connect a character with BiM, then it won't ever even if you try 100s times. 

But when you "fix" the problem like I explained and reequip the same stuff, then you can connect again with BiM. 
I suspect there're some time related mecanism the triggers this problem.

Also it doesn't always happen with the same character. So it probably doesn't come from a specific item. 

I wonder if it's not related with "ObjectModifiedMessage". 

Also it's an old bug, not new with 2.11 or even 2.10.

**Comment by Timorem** [comment]
I believe that the server close the connection and not BiM, so it's a malformated message. But I'm not sure
